content/introduction/use-cases/data-and-model-quality.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Data and Model Quality Control
+---
+
+# Problem
+
+* Are you in the business of creating data and models? 
```

Comment:
Just an idea, most of these say Dolt can do Z, Y, Z, but they leave it to the reader to understand why you would use Dolt compared other products that do X, Y, Z. One angle is maybe "this is a list of products a typical company might use to solve this set of problems; here is what the workflow looks like in that world; vs Dolt bundles the whole package, hosted, offline, backups, UX, OLTP/OLAP perf/you sacrifice Y and Z doing it the old way..."

Reply:
This is a good idea. I'll ponder it. Maybe a section called "What Dolt replaces?"